Debug-end-to-end-tests.md

```diff
@@ -0,0 +1,106 @@
+## Table of contents
+
+* [Introduction](#introduction)
+* [Using the debugger](#using-the-debugger)
+* [Rerunning with SSH](#rerunning-with-ssh)
+* [Downloading screenshots](#downloading-screenshots)
+* [Downloading screen recordings](#downloading-screen-recordings)
```

Comment:
Let's update the table of contents with the new changes and make sure all the links are working :)

Reply:
done